packages/jellyfish-transaction/src/buffer/buffer_composer.ts

```diff
@@ -393,6 +393,28 @@ export abstract class ComposableBuffer<T> implements BufferComposer {
     }
   }
 
+  /**
+   * VarUInt sized hex string, encoded in BE order buffer.
+   * String is always BE, as Javascript is uses BE by default.
+   *
+   * @param getter to read BE ordered String and write as BE ordered Buffer
+   * @param setter to read BE ordered Buffer and set as BE ordered String
+   */
+  static varUIntHexBE (getter: () => string, setter: (data: string) => void): BufferComposer {
```

Comment:
```suggestion
  static varUIntHex (getter: () => string, setter: (data: string) => void): BufferComposer {
```

Since you do not convert LE/BE order it should just be varUIntHex, it's pretty similar to the snipper below with the addition of varUInt

https://github.com/DeFiCh/jellyfish/blob/028a29459be7a323ee6eece07ebe773c692ca8c2/packages/jellyfish-transaction/src/buffer/buffer_composer.ts#L186-L195

Reply:
I was thinking if we ever need a `varUIntHexLE`. only in LE, the reverse should happen. but yeah for now we can rename it to `varUIntHex`